kedro/framework/cli/starters.py

```diff
@@ -269,6 +275,7 @@ def new(  # noqa: PLR0913
     checkout: str,
     directory: str,
     example_pipeline: str,  # This will be True or False
```

Comment:
```suggestion
    example_pipeline: str,
    telemetry_consent: str,
```
It seems the comment is outdated. Previously, the `example_pipeline` was returned with Click preprocessed to contain True/False values. However, it now retains its original 'y'/'no' values. It would be best to remove this comment.